docs/application-connector/docs/01-02-application-connector-external-apis.md

```diff
@@ -0,0 +1,24 @@
+---
+title: APIs exposed by the external solution
+
```

Comment:
Add this metadata entry to this and the 01-01 file:
```
type: Overview
```

The other option is to merge this document to the 01-01 as the separate ``## APIs exposed by the external solution`` heading.

Reply:
Done